Cargo.toml

```diff
@@ -18,10 +18,10 @@ members = ["xtask"]
 
 [dependencies]
 atty = "0.2"
-# Pinned until further action to migrate to newer clap, see https://github.com/clap-rs/clap/issues/3822#issuecomment-1154069623
-clap = { version = "~3.1", features = ["derive"] }
+clap = { version = "3.2.2", features = ["derive", "unstable-v4"] }
```

Comment:
Not sure we need this soon, check https://github.com/epage/clap/commit/7cb5bba81132a8840d5ebaba259201d35abe725f. They've changed it so people can opt-in to deprecations which should be released shortly.

Reply:
I think it's fine to keep it, when the feature is removed it'll be 4.0, since removing feature flags is breaking behaviour (requiring a major bump)